src/js/menuItem.js

```diff
@@ -50,14 +50,14 @@ export function renderMenuItems(filterTag = '') {
     container.innerHTML = '';
 
     menuData.forEach(item => {
-        if (filterTag && !item.tags.includes(filterTag)) return;
+        if (filterTag && !item.tagArray.includes(filterTag)) return;
 
         const card = document.createElement('article');
     card.classList.add('menu-product-card');
         card.innerHTML = `
```

Comment:
Net zoals eerder, liever geen `innerHTML` gebruiken: https://www.geeksforgeeks.org/javascript/what-is-the-disadvantage-of-using-innerhtml-in-javascript/